### ISSUE TITLE
Secure connections

### ISSUE THREAD

**Issue by tfrothingham-yottaa** [andnyb/sonar-slack-pusher#4]
First of all thank you, this is a great idea.
Would it be possible to add in the ability to hit up sonarqube on an https connection?  Currently it looks like an http connection is expected.

Thanks!

**Comment by andnyb** [comment]
Ok, taking it as it does not work with a SonarQube instance running https. I do not have access to an https instance to test with but is should be as simple as setting the 'Sonar root URL' to an https://address. It is implemented to run with a non public SQ and projects that has browse permissions enabled and hence not needing any specific credentials to be read.

**Comment by tfrothingham-yottaa** [comment]
Fair enough.  Do you mind if I fork your project and add in support for https?

**Comment by andnyb** [comment]
Sure, np and lets create a pull request if you sort it out.

**Comment by tfrothingham-yottaa** [comment]
Sure thing, going to be a few days before I get back around to it.
